log-collector-script/linux/eks-log-collector.sh

```diff
@@ -446,6 +447,13 @@ get_ipamd_info() {
   ok
 }
 
+get_multus_info() {
+  try "collect Multus logs"
```

Comment:
I think we should try to collect Multus logs only if its installed, similar to what we do for SELinux , is it possible to add that check here ?